saleor/product/migrations/0160_populate_product_datetimes.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 3.2.12 on 2022-02-09 15:03
+
+from django.db import migrations
+from django.db.models import F
+from django.utils import timezone
+
+
+def populate_products_datetimes(apps, _schema_editor):
+    Product = apps.get_model("product", "Product")
+
+    Product.objects.filter(updated_at__isnull=False).update(created=F("updated_at"))
+    Product.objects.filter(created__isnull=True).update(created=F("updated_at"))
```

Comment:
Shouldn't be like that. Your solution move `null` from `updated_at` field to `created` field.
```suggestion
    Product.objects.filter(created__isnull=True).update(updated_at=timezone.now())
```

Reply:
I'll change this to `Product.objects.filter(created__isnull=True).update(created=timezone.now())`